addons/html_builder/static/src/core/composite_action_plugin.js

```diff
@@ -21,6 +21,9 @@ export class CompositeActionPlugin extends Plugin {
 class CompositeAction extends BuilderAction {
     static id = "composite";
     static dependencies = ["builderActions"];
+    setup() {
+        this.loadOnClean = {};
+    }
```

Comment:
loadOnClean is a boolean
what do you think of assigning it in the constructor?
```suggestion
    loadOnClean = true;
```

Reply:
You're right.